lib/routes/admin-api/feature.js

```diff
@@ -52,6 +61,42 @@ class FeatureController extends Controller {
         }
     }
 
+    async listTags(req, res) {
+        const name = req.params.featureName;
+        const tags = await this.featureTagStore.getAllTagsForFeature(name);
+        res.json({ version, tags });
+    }
+
+    async addTag(req, res) {
+        const { featureName } = req.params;
+        const { value, type } = req.body;
+        const userName = extractUser(req);
+        try {
+            await nameSchema.validateAsync({ name: featureName });
+            await tagSchema.validateAsync({ value, type });
+            const tag = await this.featureTagStore.tagFeature({
+                value,
+                type,
+                feature_name: featureName,
```

Comment:
it feels wrong to expose the snake_case here. I would just have it `featureName`and do the db column mapping in the store. The controller should not need to know db-internals.

Reply:
Agreed. I'll change it

Reply:
Fixed in 8def623